### ISSUE TITLE
#1 Supervised learning and unsupervised learning (지도학습과 비지도학습)

### ISSUE THREAD

**Issue by Jo-won** [Jo-won/self-study-questions#1]
First of all , these cases are a range of machine learning. 

Maching learning was defined by Mitchell, "A computer program is said to learn from experience E with reespect to some class of tasks T and performance measure P, if its performance at tasks in T, as measured by P, improves with experience E."

Likewise we can say that if through learning set and validation set machine learns data and is validated by test case, it's machine learning.

In this concept, there are mainly two contents.

One of this is 'Supervised learning.'

Supervised learning go through two processes.

In first process, there are training steps. Once we collect many input images, we can abstract a number of feature vector. After building learning set by showing feature vectors in feature space, we distinct this as classifier and machine can learn to classifier. (CNN, SVM, AdaBoost, random forest...)

This classifier uses for recognition step. Feature vectors are abstracted in input image and this is classified by classifier. After all we can know what this image is classified.

In the process of modeling this classifier, we also evaluate performance of that. If we satisfy in performance by validation set, stop the learning, otherwise learn again. Criteria of performance isn't accuracy always. In fact, generation determines criteria of performance.

The other of contents is 'Unsupervised learning.'

In supervised learning, sample of this has (x,t) data. x is feature vector, t is classification. But if t not exists, this is called unsupervised learning. It doesn't have t, so it use clustering of sample.

In addition, there is 'semi-supervised learning' between supervised learning and unsupervised learning,

**Comment by Jo-won** [comment]
한글판은 축구보고..

**Comment by nhk9680** [comment]
프로필 사진 멋있어요!
